instrumentation/metric/registry.go

```diff
@@ -15,13 +18,20 @@ type Factory interface {
 type Registry interface {
 	Factory
 	String() string
-	ExportAll() map[string]interface{}
+	ExportAll() map[string]exportedMetric
```

Comment:
do we need `ExportAll` if we have `ExportAndRotate`?

Reply:
`ExportAll` is for http responses, `ExportAndRotate` is for reporting to logs. We don't want to rotate on every http request.

Reply:
so maybe rename it to reflect that it is a "peek" operation

Reply:
I have removed `ExportAndRotate` all together.